src/components/Footer/s.module.scss

```diff
@@ -86,6 +86,12 @@
     transform: rotate(360deg);
   }
 
+  @media (prefers-reduced-motion: reduce) {
```

Comment:
this is cool, how do i turn on reduce motion?

Reply:
<img width="780" alt="Screenshot_2019-11-20_at_8_24_35_AM" src="https://user-images.githubusercontent.com/1461498/69200945-5c895180-0b6f-11ea-8a39-10e922497ef0.png">

When reduced motion turned on, instead of spinning, the ghosts will just gently grow a bit.
Tangentially related blog post: https://www.chenhuijing.com/blog/the-one-which-is-impossible-to-spell